### ISSUE TITLE
build fails when I build the source code

### ISSUE THREAD

**Issue by yihichan** [botfront/botfront#806]
⚠️ Before posting a new issue, please make sure that your issue hasn't already been resolved in another issue.

**Describe the bug**
when I build the botfront source code ,there were some problems ,and the command is "meteor npm run start:docker-compose.dev"

![image](https://user-images.githubusercontent.com/17567936/97959801-144c5900-1deb-11eb-96a1-be6d1ea01123.png)

**To Reproduce**

I installed it entirely according to official instructions, and It went well until the last command.
![image](https://user-images.githubusercontent.com/17567936/97960003-83c24880-1deb-11eb-957d-72013f01acd4.png)

So,what happened and what should I do? please!

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by pheel** [comment]
@yihichan Could you provide the logs at `~/.npm/_logs/2020-11-03.....log`?

**Comment by yihichan** [comment]
yes , there seems to be something wrong with the mongo,but the mongo in docker is running well and can access mongo database 

[2020-11-03T07_33_43_391Z-debug.log](https://github.com/botfront/botfront/files/5481420/2020-11-03T07_33_43_391Z-debug.log)

**Comment by znat** [comment]
I think Meteor just doesn't allow to run as root, unless you set the env variable or use --allow-superuser option

**Comment by yihichan** [comment]
@znat  yes, great ! it work! I used command "meteor  npm run start:docker-compose.dev --allow-superuser" , and it doesn't work, but I try to change the command as "meteor --allow-superuser npm run start:docker-compose.dev"  , it work ! Thank you ever so much!
But 
another problem is comming 
![image](https://user-images.githubusercontent.com/17567936/98089747-277c2900-1ebe-11eb-8fe1-20ef11d180b6.png)

I  have edit the botfront.yml file, and  change the root_url to the machine IP address  and 8888 ,5005,5055,8000,27017,8080,3000 port is set to the public,, and I can access all the url in this project but "http://101.32.184.76:3000/project/bf/dialogue"

**Comment by MatthieuJnon** [comment]
Are you specifying the port too in your ROOT_URL variable ? what are you setting it as ?

**Comment by yihichan** [comment]
Yes ,I set ROOT_URL in  botfront.yml , as follows :
"env:
  bf_project_id: bf
  bf_url: 'http://botfront:3000/graphql'
  root_url: 'http://101.32.184.76:8888'
  augmentation_factor: 50
  mongo_url: 'mongodb://mongo:27017/bf'
version: 0.27.3
"
This file exists in both the project and the source code , I set both
